plugins/main/bridge/bridge_test.go

```diff
@@ -178,13 +181,10 @@ var _ = Describe("bridge Operations", func() {
 			links, err := netlink.LinkList()
 			Expect(err).NotTo(HaveOccurred())
 			Expect(len(links)).To(Equal(3)) // Bridge, veth, and loopback
-			for _, l := range links {
-				if l.Attrs().Name != BRNAME && l.Attrs().Name != "lo" {
-					_, isVeth := l.(*netlink.Veth)
-					Expect(isVeth).To(Equal(true))
-				}
-			}
+			link, err = netlink.LinkByName(result.Interfaces[0].Name)
 			Expect(err).NotTo(HaveOccurred())
+			_, isVeth := link.(*netlink.Veth)
```

Comment:
gomega's matcher _BeAssignableToTypeOf(expected interface)_ can be used here. Roughly

```Go
Expect(link).To(BeAssignableToTypeOf(netlink.Veth))
```

I'm not sure about the pointer aspect here.